plugins/cross-tab-copy-paste/test/index.html

```diff
@@ -12,6 +12,10 @@
 </head>
 
 <body>
+  <p>This plugin can be tested by opening up multiple tabs of this page
+    and copy/pasting the block from one tab to another. Keep in mind you can
+    not copy a block from this workspace and paste into another plugin page.
+  </p>
```

Comment:
If someone has better wording/explanation for this I am definitely open to suggestions.

Reply:
"You can only paste into a workspace that also has this plugin enabled." ?

Reply:
Perfect, Thanks!